### PR TITLE
Frontend for Donation Form

### PR DESCRIPTION
**Pull request by calistatan03**
Completed basic frontend for donation form. 

Have not completed logic for displaying preview image after image upload, will complete that soon!

### CODE REVIEW

File: app/donate/page.tsx
Comment (asdfghjkxd) [inline app/donate/page.tsx:89]: Is this console log statement supposed to be here?

File: app/donate/page.tsx
Comment (asdfghjkxd) [inline app/donate/page.tsx:137]: Once we merge the header component created by @KatyayaniPande, we can probably replace this part with the component for consistency.

File: app/donate/page.tsx
Comment (asdfghjkxd) [inline app/donate/page.tsx:167]: Should we look into standardising the overall styles for buttons?

The Button component provided by shadcn can be modified to fit in with the styles we want our buttons to look like.

**Review by asdfghjkxd — CHANGES_REQUESTED** [review]
LGTM! Once the console log statement is resolved, we can merge this PR!

File: app/donate/page.tsx
Comment (calistatan03) [reply app/donate/page.tsx:89]: Oops no need to be there! Just added in to check that my data is correct when testing the submit button, I'll remove this tomorrow after I push my new donation form fields too

File: app/donate/page.tsx
Comment (calistatan03) [reply app/donate/page.tsx:167]: Ooh yes we should standardise that...I'll change to use the Button component by shadcn! 
maybe we can use black for any submit buttons (which i think we're all using right now) and possibly green for those that involve toggling (like my cooked vs non-cooked one, and your admin/beneficiary/donor one) ? what do u think

File: app/donate/page.tsx
Comment (asdfghjkxd) [reply app/donate/page.tsx]: Ah okay, thank you!

File: app/donate/page.tsx
Comment (asdfghjkxd) [reply app/donate/page.tsx:175]: Yup sounds good! I will see what I can do to change the colour of the tabs to green!

File: app/donate/page.tsx
Comment (asdfghjkxd) [reply app/donate/page.tsx:145]: @calistatan03 The Header component has been merged into main and merged into this PR, if you wish, you could begin working on replacing this with the new Header component!

**Review by asdfghjkxd — APPROVED** [review]
LGTM! We can probably integrate the new Header component in a future PR to integrate the pages.

**Pull request merged**
